src/LinAlg/ReSolve/RefactorizationSolver.cpp

```diff
@@ -58,7 +58,6 @@
 #include "RefactorizationSolver.hpp"
 
 #include "hiop_blasdefs.hpp"
```

Comment:
@pelesh Guess you will remove this line later, once ReSolve is published as a standalone package. Maybe you'd like to address this issue in a later PR.

Reply:
Thanks for catching this. I don't think this include is needed at all.